.github/test-matrix.json

```diff
@@ -0,0 +1,110 @@
+{
+  "include": [
+    {
+      "modules": [
+        "core/trino-main"
+      ]
+    },
+    {
+      "modules": [
+        "testing/trino-tests"
+      ]
+    },
+    {
+      "modules": [
+        "plugin/trino-raptor-legacy"
+      ]
+    },
+    {
+      "modules": [
+        "plugin/trino-accumulo"
+      ]
+    },
+    {
+      "modules": [
+        "plugin/trino-cassandra"
+      ]
+    },
+    {
+      "modules": [
+        "plugin/trino-clickhouse"
+      ]
+    },
+    {
+      "modules": [
+        "plugin/trino-hive",
+        "lib/trino-orc"
+      ]
+    },
+    {
+      "modules": [
+        "plugin/trino-hive",
+        "lib/trino-parquet"
```

Comment:
These were manually colocated to keep number of jobs reasonable.
Hive module depends on ORC and Parquet module the same way Iceberg does. And both depend on some other lib modules.
Does it still make sense to keep Hive and ORC together?

Reply:
I don't think I can answer this. GIB will correctly identify modules with changes and their dependencies, so most likely we can rearrange these, but you'd have to give me specific suggestions (or add a commit).

Reply:
This is just a list of modules that'll be tested in a single job. I think the only requirement is the total job duration, and that we make sure to mention every module at least once, for those excluded in `test-other-modules`.